docs/secure-tunnels/ngrok-agent/reference/changelog.md

```diff
@@ -4,6 +4,9 @@ title: Agent Changelog
 
 # ngrok Agent Changelog
 
+### ngrok Agent 3.3.2 - \[2023-07-27\]
+* Added support for the Datadog event destination in `ngrok api`
```

Comment:
We should show an example or link to the documentation so users can find out how to use this

Reply:
good catch! let me figure out how to get that into the docs since it's not there yet